x-pack/plugins/ingest_manager/common/types/epm.ts

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
We'll need to be a bit careful here as this file is being ported over in the [server PR too](https://github.com/elastic/kibana/pull/56854/files#diff-a38b74c1d631afdd3b8e939d42ccf973R1), my comments for this file would be the same as the one I've left on that PR 😊